docs/config.rst

```diff
@@ -10,11 +10,15 @@ them in order with the latest one taking more priority:
 * ``c:\scrapyd\scrapyd.conf`` (Windows)
 * ``/etc/scrapyd/conf.d/*`` (in alphabetical order, Unix)
 * ``scrapyd.conf``
-* ``~/.scrapyd.conf`` (users home directory)
+* ``~/.scrapyd.conf`` (user's home directory)
 
 The configuration file supports the following options (see default values in
 the :ref:`example <config-example>`).
 
+.. note::
+    The user's home directory is not expanded for the user that may be specified
+    with the ``--uid`` option, but for the user invoking ``scrapyd``.
+    
```

Comment:
in short: i don't need to invoke scrapyd as root, but i usually check out a command's help and spotted that option and chose to go with it. then it took me a while to figure out why `scrapyd` wouldn't read the config from the `--uid` defined user's directory.

i invoke `scrapyd` just as that, which points to [`scrapyd.scripts.scrapyd_run:main`](https://github.com/scrapy/scrapyd/blob/master/setup.py#L47). if one was supposed to use another script, then that entrypoint shouldn't be defined at all.

shall i open a report that the `--help` command doesn't what it should do?